webplugin/js/app/mck-app.js

```diff
@@ -416,6 +417,23 @@ function ApplozicSidebox() {
             return false;
         }
     };
+
+
+    function preLoadLauncherIcon(widgetTheme) {
+        if(widgetTheme && widgetTheme.widgetImageLink) {
+            var img = new Image();
+            img.onload = function () {
+                var launcherInterval = setInterval(function(){
+                    if(document.getElementById("mck-sidebox-launcher")){
```

Comment:
It must be present in some variable, we can reuse that, if not we can create one, this way it wouldn't be searching in DOM multiple times for same element

Reply:
@devashishmamgain  Actually it is needed to be searched in the DOM because when this code fires for the first time `mck-sidebox-launcher` is not present because it is appended in the DOM via javascript. But as soon as it is fired for the second time then at that time, it is present there.

I'll add it in a variable but that variable needs to be inside the `setInterval` so that it can check whether it is appended in DOM or not. So the calculate will happen again when the `setInterval` fires for the second time. And in my numerous testing it never fired more than once so there won't be any major performance issues.